functions/graphql/models/Stripe.js

```diff
@@ -35,6 +56,13 @@ module.exports = class Stripe {
       )
     );
   }
+
+  /**
+   * Params customerId (string), planId(string), trial (bool)
+   * Returns Promise(subscription create event)
+   * QUESTION: What if they already have a subscription? would this add another subscription?
```

Comment:
This should check to make sure they don't have a subscription. If they already have one, it should throw an error.